### PR TITLE
add: stardust-non-spatial-input

### PR DESCRIPTION
**Pull request by Owen-sz**

### CODE REVIEW

**Comment by madonuko** [comment]
You accidentally pushed this to #2056.

**Comment by madonuko** [comment]
Please apply the changes requested at 2056

**Comment by Owen-sz** [comment]
> Please apply the changes requested at 2056

Look good now? Pretty sure that's what you wanted

File: anda/stardust/non-spatial-input/stardust-non-spatial-input.spec [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Tools you can easily snap together to get non-spatial input into stardust.
+
+%prep
+%autosetup -n non-spatial-input-%commit
+%cargo_prep_online
+
+%build
+
+%install
+cd non-spatial-input
```
Comment (madonuko) [inline anda/stardust/non-spatial-input/stardust-non-spatial-input.spec]: no such file or directory

your choices are `simular`, `manifold`, `display`, `azimuth`

Comment (Owen-sz) [reply anda/stardust/non-spatial-input/stardust-non-spatial-input.spec]: Done, looks like it may not like the parentheses?

**Comment by madonuko** [comment]
```
File not found: /builddir/build/BUILD/stardust-non-spatial-input-20240824.5ac7f04-build/BUILDROOT/usr/bin/display
```

Owen you can fix this

**Review by madonuko — DISMISSED** [review]

**Review by madonuko — APPROVED** [review]

**Review by madonuko — APPROVED** [review]

**Comment by raboneko** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|f39|[](https://github.com/terrapkg/packages/pull/2081)|
|✅|f40|[](https://github.com/terrapkg/packages/pull/2083)|
|✅|f41|[](https://github.com/terrapkg/packages/pull/2084)|
|✅|el9|[](https://github.com/terrapkg/packages/pull/2086)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport) and see the [Github Action logs](https://github.com/terrapkg/packages/actions/runs/10943045391) for details

**Pull request merged**